src/main/target/FATSHARK101/config.c

```diff
@@ -0,0 +1,127 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+
+#include <platform.h>
+
+//#ifdef USE_TARGET_CONFIG
+
+#include "common/axis.h"
+
+#include "fc/config.h"
+#include "fc/rc_modes.h"
+#include "fc/controlrate_profile.h"
+#include "fc/rc_adjustments.h"
+
+#include "flight/mixer.h"
+#include "flight/pid.h"
+#include "sensors/boardalignment.h"
+#include "drivers/serial.h"
+#include "io/serial.h"
+#include "io/ledstrip.h"
+#include "rx/rx.h"
+
+
+void targetConfiguration(void)
+{
+	//Set the UART2 as RX serial port
```

Comment:
Your indentation is out of whack. Please fix, and use 4 spaces instead of tabs.

Reply:
Ok, sorry for that, I will change the editor settings.
It is not a 100% clone of it as there is a new feature added to switch the camera to Black and White when the battery low warning triggers... that will be added later with some other features.